improver_tests/standardise/test_RegridLandSea.py

```diff
@@ -70,10 +70,15 @@ class Test_process(IrisTest):
 
     def setUp(self):
         """Set up input cubes and landmask"""
+        # Set domain_corner to define a lat/lon grid which overlaps and completely covers the UK standard grid,
+        # using 15.9,-2.5 as centre (as equalarea grid is configured)
```

Comment:
Should the 15.9 in this comment be 54.9 to match the code?

Reply:
It should, thanks for spotting that!

Updated in commit f10eba235deb67f993ee2bd6c2273f37cde993c0.